src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1139,9 +1140,15 @@ private void checkDockerIgnore(long startTime, File dockerBuildContext) {
             } catch (IOException e) {
                 buildContextPath = dockerBuildContext.getAbsolutePath();
             }
-            warn("The docker build command is slower than expected. You may increase performance by adding " +
-                "unneeded files and directories such as any Liberty runtime directories to a .dockerignore file in " +
-                buildContextPath + ".");
+            message += " You may increase performance by adding unneeded files and directories " +
+                "such as any Liberty runtime directories to a .dockerignore file in " +
+                buildContextPath + ".";
+        }
+        if (hasFeaturesSh.get()) {
+            message += " The RUN features.sh command is detected in the Dockerfile and this may increase build time significantly.";
```

Comment:
This message sounds as if the user is doing something wrong.  Especially if it started off with "The docker build command is slower than expected."  And since this message appears after the command has finished, should we just avoid printing the message in this case?

Reply:
I thought we should explain why the command is taking so long. I understand it happens every time but it is so slow it bears repeating. If they use cached features hopefully the whole thing finishes in under 30s. Are you saying that if they use RUN features.sh then we should not print the .dockerignore warning/suggestion at all?

Reply:
Sample message with both:
```
[WARNING] The docker build command took longer than 30 seconds. You may increase performance by adding unneeded files and directories such as any Liberty runtime directories to a .dockerignore file in /Users/paulg/WAS/devExperience/demo-devmode. The RUN features.sh command is detected in the Dockerfile and extra time may be necessary for installing features.
```